tsconfig.json

```diff
@@ -12,7 +12,8 @@
     "newLine": "lf",
     "allowSyntheticDefaultImports": true,
     "forceConsistentCasingInFileNames": true,
-    "strict": true
+    "strict": true,
+    "skipLibCheck": true
```

Comment:
Without this option, the `tsc` command would complain about typing inside `.ts` files inside the `node_modules` directory.

Reply:
This is fixed in https://github.com/svg/svgo/releases/tag/v3.0.1

Reply:
Thank you for the notice!
It now gives
```
node_modules/.pnpm/svgo@3.0.1/node_modules/svgo/plugins/plugins-types.ts:1:28 - error TS7016: Could not find a declaration file for module 'csso'. '/mnt/data/WorkingDirectory/Git/dargmuesli/cssnano/node_modules/.pnpm/csso@5.0.5/node_modules/csso/cjs/index.cjs' implicitly has an 'any' type.
  Try `npm i --save-dev @types/csso` if it exists or add a new declaration (.d.ts) file containing `declare module 'csso';`

1 import type * as csso from 'csso';            
```
Is that something that should be fixed by cssnano or svgo?

Reply:
You need to add `@types/csso` to dev dependencies